graf2d/asimage/src/libAfterImage/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# libAferImage CMakeLists.txt
+
+PROJECT(AFTERIMAGE)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.0)
```

Comment:
Quick question, should we put here the minimum CMake version as it is in ROOT?
e.g. 
```
if(MSVC)
  # required for the following feature & bug fix:
  # 3.15: Added $<REMOVE_DUPLICATES:list> generator expression
  # 3.16: Bug fix with CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS: the auto-generated exports
  #       are now updated only when the object files providing the symbols are updated
  cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
else()
  cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
endif()
```

Reply:
Oh right, I don't even know why there is a minimal version here, but I agree, I'll set it to 3.16, thanks @oshadura !